src/components/Book.js

```diff
@@ -1,21 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
 
-const TR = styled.tr`
-  background: blue;
-  color: white;
-  text-align: center;
-  width: 100%;
-`;
-
-export default function Books({ book }) {
+export default function Books({ book, removeBook }) {
   return (
-    <TR>
-      <td>{book.id}</td>
-      <td>{book.title}</td>
-      <td>{book.category}</td>
-    </TR>
+    <tr claasName="erow" key={book.id}>
+      <td className="trow">{book.id}</td>
+      <td className="trow">{book.title}</td>
+      <td className="trow">{book.category}</td>
+      <td className="trow">
+        <button className="btn3" type="submit" onClick={() => removeBook(book)}>
+          Remove Book
+        </button>
```

Comment:
This should be implemented in `BookList` component.
```suggestion
        <button className="btn3" type="submit" onClick={removeBook}>
          Remove Book
        </button>
```